pkg/loki/loki.go

```diff
@@ -772,6 +805,10 @@ func (t *Loki) setupModuleManager() error {
 		All: {QueryScheduler, QueryFrontend, Querier, Ingester, PatternIngester, Distributor, Ruler, Compactor, UI},
 	}
 
+	if t.Cfg.IngestLimits.Enabled {
+		deps[All] = append(deps[All], IngestLimits, IngestLimitsFrontend)
+	}
```

Comment:
Is this necessary anymore beyond local testing? The `All` target includes the partition ingester in the single binary so limits are enforced in one place. IIRC I added this line on the feature branch to make local testing easy.

Reply:
I think we might want to keep this for now for local testing in single binary mode. We can remove it later?